src/components/IOBanner.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import { css } from '@emotion/react';
+import useMobileDetect from 'use-mobile-detect-hook';
+import { Button, Link } from '@newrelic/gatsby-theme-newrelic';
+import ioBanner from '../images/ioBanner.svg';
+
+const IOBanner = () => {
+  const detectMobile = useMobileDetect();
```

Comment:
This could be moved into a prop since the instant-observability page is showing/hiding based on the same hook! Up to you though

Reply:
true! it's kinda nice if it can take care of itself if it got used elsewhere (though what are the chances 😂) but a prop seems good in this situation